go-controller/pkg/ovn/policy.go

```diff
@@ -370,7 +371,7 @@ func (oc *Controller) setNamespaceACLLogging(ns string, nsInfo *namespaceInfo, a
 func (oc *Controller) getNamespaceACLs(ns string, nsInfo *namespaceInfo) []string {
 	klog.V(5).Infof("Looking for acls on ns: %s", ns)
 	var allPolicies []*gressPolicy
-	for _, policy := range nsInfo.networkPolicies {
+	for _, policy := range getPoliciesInDeterministicOrder(nsInfo.networkPolicies) {
```

Comment:
there's 3 ways this can go:
  - remove the test with multiple policies (not good, but doesn't impact the production code... not very fond of this sort).
  - sort these (which allows for the test w/ multiple policies to be deterministic)
  - add a boolean to the `controller` that the `FakeOVN` would use to indicate it needs these to be sorted.

A better solution would be a way to indicate the inner mocking that it can accept either one of: 
```
... ovn-nbctl list acl policy1 policy2
... ovn-nbctl list acl policy2 policy1
```

I have no clue if the above is possible.

Reply:
I'm now using N `find`s (one for each policy) instead to avoid reaching this issue.